test/SlowTests/MailingList/Tobias.cs

```diff
@@ -1,186 +0,0 @@
-// -----------------------------------------------------------------------
-//  <copyright file="Tobias.cs" company="Hibernating Rhinos LTD">
-//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
-//  </copyright>
-// -----------------------------------------------------------------------
-
-using System.Linq;
-using FastTests;
-using Lucene.Net.Analysis;
-using Raven.Client;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Session;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.MailingList
-{
-    public class Tobias : RavenTestBase
-    {
-        [Fact(Skip = "Missing feature: Collation")]
```

Comment:
This is basically creating a ton of analyzers for no real benefit. Users can just define a custom analyzer on their own.